src/components/ActionPanel/index.tsx

```diff
@@ -14,7 +14,7 @@ import Transactions from '../Transactions';
 import Loader from '../Loader';
 import AccountStatus from '../AccountStatus';
 
-import { Send, Receive, Invoke, CodeStorage, WithdrawDeposit, Delegate } from '../TabContents';
+import { Send, Receive, Invoke, CodeStorage, Withdraw, Deposit, Delegate } from '../TabContents';
```

Comment:
The folder structure for contract UIs should be: `/src/contracts/`. The code here should be in `/src/contracts/BabylonDelegation`. That directory should contain all these files plus a container. The container will host the tabs instead of having `ActionPanel` include them one by one.